app/services/payout_report_publisher_includer.rb

```diff
@@ -6,7 +6,8 @@ def initialize(payout_report:, publisher:, should_send_notifications:)
   end
 
   def perform
-    return if !@publisher.has_verified_channel? || @publisher.excluded_from_payout?
+    return if !@publisher.has_verified_channel? || @publisher.locked? || @publisher.excluded_from_payout?
+    publisher_has_unsettled_balance = false
```

Comment:
Extra non-intended line?

Reply:
Fixed, will keep some spacing for readability